.env.default

```diff
@@ -6,4 +6,5 @@ AWS_S3_ENDPOINT_URL=http://localhost:9090
 AWS_DB_REGION_NAME=eu-central-1
 AWS_DB_TABLE_NAME=test-db
 AWS_DB_ENDPOINT_URL=http://localhost:8080
-KML_STORAGE_HOST_URL=http://localhost:9090/test-bucket
\ No newline at end of file
+KML_STORAGE_HOST_URL=http://localhost:9090/test-bucket
+ALLOWED_DOMAINS=.*localhost,.*admin\.ch,.*bgdi\.ch
```

Comment:
I understood from recent PR, that we use .* in `.env.default` as a default value here?

Reply:
Yeah but I noticed that if we want to run the e2e tests on localhost, we do in e2e tests check the origin header based on real value, so if we configure the service with `.*` then e2e test are failing. I would leave the default value in service to `.*` but set the variable to a correct value with the env file when working locally, this simplify our life for testing and debugging.

Reply:
OK, I see, thanks. Then this would need to be changed for some other services, too, I guess..